### ISSUE TITLE
Android setValue and touchEnd

### ISSUE THREAD

**Issue by kgividen** [mpociot/TiCircularSlider#17]
Would anyone be interested in implementing setValue and touchEnd for Android like Hans did for iOS?  I'd be willing to put some money towards it.

**Comment by manumaticx** [comment]
@kgividen the `touchend` event is inherited from Ti.UI.View and should already work. Adding getter / setter for the value property is a simple change, I can do this. As soon as I'm able to test this, I'll send a PR.

**Comment by kgividen** [comment]
@manumaticx the touchend fires but it doesn't pass the current value into the event.  That's awesome if you're able to update this.  It will be great for a project I'm currently doing.  Thanks!

**Comment by kgividen** [comment]
@manumaticx any chance you've been able to look at this?  Thanks.

**Comment by manumaticx** [comment]
@kgividen yes but I'm struggling with a bug that appeared after compiling with 5.1.2.GA 
Sorry, but I was a little busy last week. I'll give it another show tomorrow.

**Comment by manumaticx** [comment]
I'm sorry but I just can't get it working as expected. My [current progress](https://github.com/manumaticx/TiCircularSlider/commit/df788966566a1d88a20daa1e239d9722144be933) throws an Error like this 
```
java.lang.NoSuchMethodError: no non-static method "Lde/marcelpociot/circularslider/ViewProxy;.setValue(I)V"
```
I need to investigate later

**Comment by kgividen** [comment]
Ugg..that doesn't sound good.  I need to see if I can get a setup put together and look at these modules.  I can code in Java and have done some limited android coding but haven't coded a module yet for titanium.  Then maybe I could help.

Is that error because setValue doesn't exist in the native module?  it doesn't seem like that's the case because it's referencing the marcelpociot/circularslider which makes it sound like it's on the Ti side of things.

**Comment by kgividen** [comment]
@manumaticx I've been playing with this a bit but this is new to me.  I compiled your code via ANT and I'm using 5.1.1.GA.  It looks like on the handleSetProperty method you set the value passed in as a property value.  

When are you getting the error?  This method seems to work and I don't see an exception but truthfully I'm not sure I've got my system set such that I would.

It does look like that property is getting set on the view though because when I get the touchend event back e.source.value is set to what I set it to in the setValue.

However, it doesn't look like it's totally working because the setProgress for the native module doesn't seem to fire or at least update the slider to the current value visually...

You might already know all this and this module process is still a little new to me so I'm just working through it.
